.flake8/azul_flake8.py

```diff
@@ -305,6 +305,58 @@ def run(self):
         return visitor.errors
 
 
+class AzulLines:
+    name = 'azul_lines'
+    version = 1.0
+
+    begin_skip = '# noqa lines: begin'
+    end_skip = '# noqa lines: end'
+
+    def __init__(self, tree, lines):
+        # We need tree as a parameter otherwise run() won't be called
+        self.lines = lines
+
+    def run(self):
+        errors: List[ErrorInfo] = []
+        for num, line in self.unskipped_lines():
```

Comment:
nit.
s/num/line_num/
s/line/line_contents/

on line 349 we use line=num, which is slightly confusing.

Reply:
I'm changing `num` to `line_num` but leaving `line` as `line` otherwise some lines will require ugly reformatting :joy: